### ISSUE TITLE
Misleading explanation

### ISSUE THREAD

**Issue by PfisterAn** [vouch/vouch-proxy#451]
It appears like the following is a misleading explanation, these domains are actually referring to Email-Domains, so instead of whitelisting individual users its for whitelisting a full domain of a specific user-base. This is not clear when looking at the explanation.

https://github.com/vouch/vouch-proxy/blob/a901c3f927d0cd1b07e2937bda98c6af57137b82/config/config.yml_example#L27

**Comment by bnfinet** [comment]
@PfisterAn for whitelisting, you should add email addresses to `vouch.whitelist`... 

https://github.com/vouch/vouch-proxy/blob/a901c3f927d0cd1b07e2937bda98c6af57137b82/config/config.yml_example#L48

If you'd like to offer an improvement to `domains` to improve the clarity or any aspect of the VP docs (which could always use some polish) we'd love to see it and I'm always happy to review doc focused PRs.

I'm going to close this but feel free to continue the discussion.

**Comment by PfisterAn** [comment]
Hi,

i think that’s the misunderstanding I also had … so I protected a specific domain with Vouch Proxy, say sub.mycompany.com and for that I added mycompany.com to vouch.domains. So far so good, when I logged in with mymail@mycompany.com I was able to get onto the site successfully, permitted by my IDP as well, obviously.
I am using Azure AD … there we also allow guest users to access the site, so I tried with mymail@gmail.com, which is also allowed by the IDP and once logged in I got onto the Bad Request (400 error page) of the vouch Proxy.
Checking the journal on my box did tell me that the gmail.com domain isn’t  allowed for login, adding gmail.com to the vouch.domains let this account work also … so essentially the vouch.domains list is rather related to whitelisting domains than anything related to the configuration around the used domain of the hosting site (which was my take when I read the comments in the example.yml file).
